ddtrace/runtime/__init__.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+
+import ddtrace
+
+
+class RuntimeMetrics(object):
+    """
+    Runtime metrics service API.
+
+    This is normally started automatically by ``ddtrace-run`` when the
+    ``DD_RUNTIME_METRICS_ENABLED`` variable is set.
+
+    To start the service manually, invoke the ``enable`` static method::
+
+        from ddtrace.runtime import RuntimeMetrics
```

Comment:
I wonder if we should bump it up a level to make it slightly more accessible:

```python
from ddtrace import RuntimeMetrics
```

Reply:
The reason why I have created the intermediate `runtime` module was to align the metrics API to the profiling one, since for that we have the `profiling` package. But I'm happy to bump it up if that's the consensus.

Reply:
I don't feel strongly about it at all and we can always add it in later!